jflac-codec/src/test/java/org/jflac/sound/spi/FlacAudioFileReaderTest.java

```diff
@@ -197,8 +197,8 @@ public void testGetAudioFileFormatWithFlacFile() throws UnsupportedAudioFileExce
         final AudioFileFormat audioFileFormat = flacAudioFileReader.getAudioFileFormat(getFlacTestFile("cymbals.flac"));
         assertNotNull(audioFileFormat);
         assertEquals("flac", audioFileFormat.getType().getExtension());
-        assertEquals(new Long(9338775), audioFileFormat.getProperty("duration"));
-        assertEquals(411840, audioFileFormat.getFrameLength());
+//        assertEquals(new Long(9338775), audioFileFormat.getProperty("duration"));
```

Comment:
Was the removal of these assertions intentional? If yes, could we replace them by equivalent assertions to keep testing the same things as before?